### DISCUSSION TITLE
Featured Artist browse in game

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#30033] [Ideas]
In the Main Menu you have a green "browse" button, which leads you to the beatmap browse.
I think it could be split like the "edit" and "play" button, and the options would be "Beatmap Listing" and "FA Listing"
It would allow for faster downloading FA map templates, and just overall browsing the FA listing in game (and if possible it could have an track search like the web, would make this even better), and just make the button more usefull, because im pretty sure everyone either clicks Ctrl-B or just the button that leads to the Beatmap Listing in the header.

**Comment by Stoppedpuma** [comment]
Regarding creation using the featured artist templates, this has already been mentioned at #18904.
